sdks/python/apache_beam/io/gcp/experimental/spannerio.py

```diff
@@ -1008,31 +1007,30 @@ def _reset_count(self):
     self._cells = 0
 
   def process(self, element):
-    mg_info = element.info
+    for elem in element:
+      mg_info = elem.info
+      if mg_info['byte_size'] + self._size_in_bytes > \
```

Comment:
This seems like a change to the implementation not part of the implementation. Probably should be a separate PR with a JIRA.

Reply:
Sure. Should I create a new Jira ticket and (1) add ticket number in this PR for reference OR (2) create a new PR for this change, and once it gets merge then I rebase this PR and request review? 

I think the first approach required less time to close the tickets! What you suggest?

Reply:
I think (2) is better but we should fix the Spanner connector implementation to work for arbitrary bundle sizes than reducing the bundle to a single element for the test.

Reply:
Since i've reverted the changes from the connector, there is no need to create new tickets. Resolving this conversation! - Thanks!